ads/gumgum.md

```diff
@@ -0,0 +1,39 @@
+<!---
+Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# GumGum
+
+## Example
+
+### Slot Ad
+```html
+<amp-ad width=300 height=100
+  type="gumgum"
+  data-zone="ggumtest"
+  data-slot="3883">
+  <div placeholder>A 300x100 placeholder</div>
```

Comment:
We provide default placeholder/fallback now. They only needs to be added if customization is needed.